### PR TITLE
Added M-method example

### PR DESCRIPTION
**Pull request by llevcono**

### CODE REVIEW

File: 17-LinProgSimplex/examples.tex [hunk 1]
```diff
@@ -520,6 +520,99 @@ \section{Задача 3}
...
+& 3x_3 + x_4 = 1\\
+& x_{1,2,3,4} \geq 0
+\end{split}
+\end{equation*}
+
+Идея заключается в том, чтобы представить задачу в виде суммы
+$$
+\sum_{j=1}^{n} c_j x_j + M \sum_{i=1}^m y_i
+$$
+где $M$ - некоторый большой положительный параметр, а $y_i$ - вспомогательные переменные из первой фазы двухфазного симплекс-метода.
```
Comment (amkatrutsa) [inline 17-LinProgSimplex/examples.tex:541]: Don't mix two phases simplex method and M-method.

File: 17-LinProgSimplex/examples.tex [hunk 2]
```diff
@@ -520,6 +520,99 @@ \section{Задача 3}
...
+
+В этой задаче мы опустим искусственную переменную $x_8$ за ненадобностью.
```
Comment (amkatrutsa) [inline 17-LinProgSimplex/examples.tex:543]: Now we do not have any $x_8$ to skip it.

File: 17-LinProgSimplex/examples.tex [hunk 3]
```diff
@@ -520,6 +520,99 @@ \section{Задача 3}
...
+ 
+\textbf{} Составим вспомогательную задачу
+\begin{equation*}
+\begin{split}
+& \min x_1 + x_2 + x_3 + Mx_5 + Mx_6 + Mx_7\\
+\text{s.t. } & x_1 + 2x_2 + 3x_3 + x_5= 3\\
+& -x_1 + 2x_2 + 6x_3 + x_6 = 2\\
+& 4x_2 + 9x_3 + x_7 = 5\\
+& 3x_3 + x_4 = 1\\
+& x_{1,2,3,4} \geq 0
```
Comment (amkatrutsa) [inline 17-LinProgSimplex/examples.tex:553]: And what about $x_5, x_6, x_7 \geq 0$?

File: 17-LinProgSimplex/examples.tex [hunk 4]
```diff
@@ -520,6 +520,99 @@ \section{Задача 3}
...
+$x_1 = 1/2$ & $1$ & $0$ & $-3/2$ & $0$ & $1/2$ & $-1/2$ & $0$\\
+$x_2 = 5/4$ & $0$ & $1$ & $9/4$ & $0$ & $1/4$ & $1/4$ & $0$\\
+$x_7 = 0$ & $0$ & $0$ & $0$ & $0$ & $-1$ & $-1$ & $1$ \\
+$x_4 = 1$ & $0$ & $0$ & $3$ & $1$ & $0$ & $0$ & $0$ \\
+\hline
+\end{tabular}
+\label{tab::simplex_36}
+\end{table}
+\newpage
+Так как при достаточно больших М все оценки замещения положительные, получено решение исходной задачи.
```
Comment (amkatrutsa) [inline 17-LinProgSimplex/examples.tex:614]: Please use $M$

**Comment by amkatrutsa** [comment]
Why do you close pull request instead of edit it?
Please, find some tutorial about pull requests like https://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request

**Comment by llevcono** [comment]
I assumed you wanted it to be done in one commit.

**Comment by amkatrutsa** [comment]
One fix - one pull-request
Also use git rebase command to squash multiple commits into one

**Comment by llevcono** [comment]
I have already made new pull request with your fixes included. 
https://github.com/amkatrutsa/MIPT-Opt/pull/37

**Comment by llevcono** [comment]
Unfortunately, I accidentally deleted this fork, thus this pull request cannot be reopened. New pull request #37 is from a new fork. Sorry for messing up the counter.

**Pull request closed without merging**